expression/builtin_string_vec_test.go

```diff
@@ -87,7 +87,9 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString}, geners: []dataGenerator{&randSpaceStrGener{10, 100}}},
 		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString, types.ETString}, geners: []dataGenerator{&randLenStrGener{10, 20}, &randLenStrGener{5, 25}}},
 	},
-	ast.LTrim: {},
+	ast.LTrim: {
+		{retEvalType: types.ETString, childrenTypes: []types.EvalType{types.ETString}, geners: []dataGenerator{&randSpaceStrGener{10, 100}}},
```

Comment:
Consider using `randSpaceStrGener` in `builtin_string_vec_test.go`, to make the generated string have a chance by starting with space.

Reply:
I have already use `randSpaceStrGener `